script/hassfest/mypy_config.py

```diff
@@ -167,7 +198,7 @@ def generate_and_validate(config: Config) -> str:
     mypy_config.add_section(components_section)
     mypy_config.set(components_section, "no_implicit_reexport", "true")
 
-    for core_module in strict_core_modules:
+    for core_module in sorted(strict_core_modules):
```

Comment:
This is not needed anymore, as it is fixed within the check of `.strict-typing`
(same applies below)